### DISCUSSION TITLE
Should discoverability of this package be improved?

### DISCUSSION THREAD

**Discussion by filipef101** [gbumps/react-native-screenguard#75] [General]
Shout out @gbumps for creating a open source library that does the same a paid SDKs. And probably better 🥇 

I only found this after much searching, almost found it by accident and then still tough it was a wrapper for a paid sdk for a moment.

Most people including me look for keywords like:  `prevent`, `screenshot`, `block`, `recording`, `screen`, `record`, `video`, `capture`, `screencast`

**Comment by gbumps** [comment]
Hi @filipef101. Thanks for your support. Very appreciated about it!

Beside, I've already put some keywords on the npm package though, also for the web document for bot search to be crawled and easily discoverable. 

I think it might take a while, but if you have any different idea to make the lib widely known to all developers around the world faster, just bring it up to the discussion and I'll carefully considered. 

Thanks!
